Libs/MRML/Core/vtkMRMLTransformDisplayNode.h

```diff
@@ -49,6 +49,7 @@ class VTK_MRML_EXPORT vtkMRMLTransformDisplayNode : public vtkMRMLDisplayNode
     VIS_MODE_GLYPH,
     VIS_MODE_GRID,
     VIS_MODE_CONTOUR,
+    VIS_MODE_INTERACTIVE,
```

Comment:
This is not an alternative visualization mode. We should be able to modify the transform regardless of visualizing it.